### PR TITLE
Authenticate Devzone Session

### PR DESCRIPTION
**Pull request by MagWeaver**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -36,6 +36,6 @@ gulp.task('default', ['buildprod'], () => {
 });
 
 gulp.task('copy', () => {
-	return gulp.src('./dist/aws-wrapper.min.js')
+	return gulp.src('./dist/aws-wrapper.js')
```
Comment (TheBosZ) [inline gulpfile.js]: Don't modify this task, create a new task.

File: src/index.ts [hunk 2]
```diff
@@ -243,6 +243,22 @@ namespace Cognito {
 			}));
 	};
 
+	export const startDevzoneSession  = async (credential): Promise<void> => {
+		AWS.config.credentials = credential;
+        await new Promise((resolve, reject) => {
```
Comment (TheBosZ) [inline src/index.ts]: Uh.... why are you awaiting this promise?

If you need to return a promise, just return the promise. No need to make it more complicated than needed.

Comment (MagWeaver) [reply src/index.ts]: I agree, wan't intentionally put there.

Comment (MagWeaver) [reply src/index.ts]: Good eye.

**Pull request merged**